### PR TITLE
Ref #13270 - remove some inline events

### LINKED ISSUE #13270: Remove inline javascript events
See more information in previous issue:
#12261 (Remove inline javascript)

Below remained places that use inline JS events:
```
./view_operations.php:        <td><input type="text" size="20" name="new_name" onfocus="this.select()"
./view_create.php:        . ' onfocus="this.select()" maxlength="64"'
./view_create.php:    . ' onfocus="this.select()"'
./view_create.php:    $htmlString .= ' onclick="selectContent(this, sql_box_locked, true)"';
./templates/columns_definitions/column_definitions_form.phtml:                               onfocus="this.select()" />
./templates/database/designer/having_query_panel.twig:                                    onclick="document.getElementById('query_having').style.display = 'none';" />
./templates/database/designer/aggregate_query_panel.twig:                                    onclick="document.getElementById('query_Aggregate').style.display = 'none';" />
./templates/database/designer/side_menu.phtml:           onclick="build_query('SQL Query on Database', 0)"
./templates/database/designer/rename_to_panel.twig:                                    onclick="document.getElementById('query_rename_to').style.display = 'none';" />
./templates/database/designer/where_query_panel.twig:                                    onclick="document.getElementById('query_where').style.display = 'none'" />
./templates/database/structure/check_all_tables.phtml:         / <a href="#" onclick="unMarkAllRows('tablesForm'); <?= $overhead_check; ?> return false;"><?= __('Check tables having overhead'); ?></a>
./templates/server/databases/table_row.phtml:        <a onclick="PMA_commonActions.setDb('<?= PMA\libraries\Sanitize::jsFormat($current['SCHEMA_NAME']) ?>');"
./templates/table/index_form.phtml:      onsubmit="if (typeof(this.elements['index[Key_name]'].disabled) != 'undefined') this.elements['index[Key_name]'].disabled = false;">
./templates/table/index_form.phtml:                       onfocus="this.select()" />
./templates/table/index_form.phtml:                                   onfocus="this.select()"name="index[columns][sub_parts][]"
./templates/table/index_form.phtml:                                   onfocus="this.select()"
./templates/table/structure/actions_in_table_structure.phtml:            <a href="#" onclick="$('input:checkbox').prop('checked', false);
./templates/table/structure/actions_in_table_structure.phtml:            <a href="#" onclick="$('input:checkbox').prop('checked', false);
./templates/table/structure/add_column.phtml:<form method="post" action="tbl_addfield.php" id="addColumns" name="addColumns" onsubmit="return checkFormElementInRange(this, 'num_fields', '<?= str_replace
./templates/table/structure/add_column.phtml:    <?php $num_fields = '<input type="number" name="num_fields" value="1" onfocus="this.select()" min="1" required />'; ?>
./templates/table/structure/add_column.phtml:    <select name="after_field" onchange="checkFirst()">
./templates/table/search/column_comparison_operators.phtml:        onchange="changeValueFieldType(this, <?= $search_index; ?>)">
./templates/table/search/zoom_result_form.phtml:                        <a href="#" onclick="displayHelp();">
./libraries/sql.lib.php:            . ' onsubmit="return ! emptyCheckTheField(this,'
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'newname\')">';
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'newname\')">';
./libraries/operations.lib.php:        . ' onsubmit="return emptyCheckTheField(this, \'new_name\')">'
./libraries/operations.lib.php:        . 'onsubmit="return emptyCheckTheField(this, \'new_name\')">'
./libraries/sql_query_form.lib.php:        $auto_sel = ' onclick="selectContent(this, sql_box_locked, true);"';
./libraries/sql_query_form.lib.php:            . 'multiple="multiple" ondblclick="insertValueQuery()">';
./libraries/sql_query_form.lib.php:                . ' value="&lt;&lt;" onclick="insertValueQuery()"'
./libraries/sql_query_form.lib.php:                . ' onclick="insertValueQuery()" />';
./libraries/display_change_password.lib.php:        . 'onclick="pma_pw.value = \'\'; pma_pw2.value = \'\'; '
./libraries/display_change_password.lib.php:        . 'onclick="document.getElementById(\'text_pma_change_pw\').focus();" '
./libraries/tracking.lib.php:        . '<option value="execution" onclick="alert(\''
./libraries/plugin_interface.lib.php:            $ret .= ' onclick="if (!this.checked &amp;&amp; '
./libraries/server_status_processes.lib.php:            $retval .= 'onmouseout="$(\'.soimg\').toggle()" '
./libraries/server_status_processes.lib.php:                . 'onmouseover="$(\'.soimg\').toggle()"';
./libraries/rte/rte_footer.lib.php:        $retval .= "onclick='$.datepicker.initialized = false;'>";
./libraries/rte/rte_routines.lib.php:    $retval .= "       class='rte_form ajax' onsubmit='return false'>\n";
./libraries/replication_gui.lib.php:    $html .= ' onsubmit="return checkAddUser(this);">';
./libraries/replication_gui.lib.php:        . '" onclick="suggestPassword(this.form)" />'
./libraries/display_export.lib.php:    $multi_values .= ' onclick="setSelectOptions'
./libraries/display_export.lib.php:    $multi_values .= ' onclick="setSelectOptions'
./libraries/display_export.lib.php:            . '" id="buttonGo" onclick="check_time_out('
./libraries/display_export.lib.php:    $html .= '" onfocus="this.select()" />';
./libraries/display_export.lib.php:    $html .= '" size="5" onfocus="this.select()" />';
./libraries/plugins/auth/AuthenticationCookie.php:                echo ' onchange="document.forms[\'login_form\'].'
./libraries/plugins/transformations/abs/DateFormatTransformationsPlugin.php:            return '<dfn onclick="alert(\'' . Sanitize::jsFormat($source, false) . '\');" title="'
./libraries/DbSearch.php:            . 'onclick="setSelectOptions(\'db_search\','
./libraries/DbSearch.php:            . 'onclick="setSelectOptions(\'db_search\','
./libraries/normalization.lib.php:        . '" onclick="goToFinish1NF();"'
./libraries/normalization.lib.php:        . '" onclick="goToStep4();"'
./libraries/server_privileges.lib.php:       . ' onsubmit="return checkAddUser(this);"'
./libraries/server_privileges.lib.php:        . 'onsubmit="return checkAddUser(this);" '
./libraries/DisplayResults.php:            . 'onsubmit="return '
./libraries/DisplayResults.php:        $onsubmit = 'onsubmit="return '
./js/functions.js:    value="' + PMA_messages.strGenerate + '" onclick="suggestPassword(this.form)" /><input type="text" name="generated_pw" id="generated_pw" /></td></tr>');
./js/functions.js:    <span class="options"><input type="button" class="button" id="button_generate_password" value="' + PMA_messages.strGenerate + '" onclick="suggestPassword(this.form)" /></span>
./js/functions.js:                    PMA_messages.pickColumn + '" onclick="autoPopulate(\'' + colid + '\',' + i + ')"/></td>';
./js/functions.js:                                PMA_messages.pickColumn + '" onclick="autoPopulate(\'' + colid + '\',' + i + ')"/></td>';
./js/pmd/history.js:                str += '<img src="' + pmaThemeImage + 'pmd/or_icon.png" onclick="and_or(' + i + ')" title="OR"/></td>';
./js/pmd/history.js:                str += '<img src="' + pmaThemeImage + 'pmd/and_icon.png" onclick="and_or(' + i + ')" title="AND"/></td>';
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_delete(' + i + ')>' + PMA_getImage('b_drop.png', 
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_edit(' + i + ')>' + PMA_getImage('b_edit.png', 
./js/pmd/history.js:                    '<td onmouseover="this.className=\'history_table\';" onmouseout="this.className=\'history_table2\'" onclick=history_delete(' + i + ')>' + PMA_getImage('b_drop.png', 
./js/normalization.js:                $('.tblFooters').html('<input type="button" onClick="processDependencies(\'\', true);" value="' + PMA_messages.strDone + '"/>');
./js/normalization.js:                $('.tblFooters').html('<input type="submit" value="' + PMA_messages.strDone + '" onclick="processDependencies(\'' + escapeJsString(escapeHtml(data.primary_key)) + 
./js/normalization.js:                        "onclick=\"$('#newCols').html('');$(this).parent().html('')\"/>");
./js/normalization.js:                        "onclick=\"$('#newCols').html('');$(this).parent().html('')\"/>");
./js/normalization.js:            $('.tblFooters').html('<input type="submit" value="' + PMA_messages.strCancel + '" onclick="$(\'#newCols\').html(\'\');$(\'#extra input[type=checkbox]\').prop(\'checked
./js/normalization.js:                '<input type="submit" value="' + PMA_messages.strGo + '" onclick="moveRepeatingGroup(\'' + escapeJsString(escapeHtml(repeatingCols)) + '\')"/>');
```

### PR DESCRIPTION
**Pull request by Thomas1664**
### Description

Added event handlers for some inline events

Related to #13270 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [] Any new functionality is covered by tests.

### CODE REVIEW

File: js/src/sql.js [hunk 1]
```diff
@@ -872,6 +872,19 @@ AJAX.registerOnload('sql.js', function () {
     $('#insertBtn').on('click', function () {
         Functions.insertValueQuery();
     });
+    
+    $('#view_as').on('click', function () {
+        Functions.selectContent(this, sqlBoxLocked, true);
```
Comment (williamdes) [inline js/src/sql.js:877]: You can add `/* global sqlBoxLocked */ // js/functions.js` at the top of the file to fix the JS lint error

**Review by williamdes — APPROVED** [review]
The diff looks good

File: js/src/sql.js [hunk 2]
```diff
@@ -872,6 +873,19 @@ AJAX.registerOnload('sql.js', function () {
...
+    });
+
+    $('#select_all').on('click', function () {
+        Functions.setSelectOptions('db_search', 'criteriaTables[]', true);
+        return false;
+    });
+    $('#unselect_all').on('click', function () {
+        Functions.setSelectOptions('db_search', 'criteriaTables[]', false);
+        return false;
+    });
```
Comment (mauriciofauth) [inline js/src/sql.js:888]: These two events (`$('#select_all')` and `$('#unselect_all')`) can be moved to `js/src/database/search.js`.

Also, add an `.off` for each `.on` that you added in the `registerTeardown` function on the same file.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Good work!

**Comment by Thomas1664** [comment]
Sign-off-by: Thomas Heinrichs

**Review by MauricioFauth — APPROVED** [review]

**Comment by williamdes** [comment]
Merged, thank you Thomas !

**Pull request merged**